src/style.css

```diff
@@ -0,0 +1,75 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+}
+
+.nav {
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin-right: 131px;
```

Comment:
Don't use 'magic numbers'. Instead of this weird margin use `justify-content: space-between` in your flex container